libs/platform/src/lib/components/form/combobox/combobox.module.ts

```diff
@@ -20,6 +20,6 @@ import { PlatformAutoCompleteModule } from '../auto-complete/auto-complete.modul
         PlatformAutoCompleteModule
     ],
     providers: [DynamicComponentService],
-    exports: [ComboboxComponent, TemplateModule]
+    exports: [ComboboxComponent, TemplateModule, HighlightPipe]
```

Comment:
I think exposing that pipe to users who will import `comboboxModule` isn't a clear solution. 

By the way we also have `SearchHighlightPipe` which has the same name `| highlight` - could them be combined? If no - won't it conflict where both modules are imported?